lang/cs/Org.Apache.REEF.IMRU/API/IPerMapperConfigs.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Collections.Generic;
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Tang.Interface;
+
+namespace Org.Apache.REEF.IMRU.API
+{
+    /// <summary>
+    /// Class containing mapper specific configuration
+    /// </summary>
+    public interface IPerMapperConfigs
+    {
+        /// <summary>
+        /// Per-mapper configurations
+        /// </summary>
+        List<IConfiguration> MapperConfigurations { get; set; }
```

Comment:
I'm not sure `List` is the right type for this. For one, it should be `IList`, but that is not my main concern: `IList` suggests that there is an order to these configurations, and that the order somehow matters, like in the various MPI calls. Hence, I think `ISet` would be a more appropriate type. Sounds good?